File: solutions/kotlin/raindrops/1/src/main/kotlin/Raindrops.kt
object Raindrops {

    fun convert(n: Int): String {
        val a = n % 3 == 0
        val b = n % 5 == 0
        val c = n % 7 == 0
        if (!a && !b && !c) {
            return n.toString()
        } else {
            var ans = ""
            if (a) {
                ans += "Pling"
            }
            if (b) {
                ans += "Plang"
            }
            if (c) {
                ans += "Plong"
            }
            return ans
        }
    }
}

File: solutions/csharp/raindrops/1/Raindrops.cs
using System.Text;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var a = number % 3 == 0;
        var b = number % 5 == 0;
        var c = number % 7 == 0;
        if (!a && !b && !c)
        {
            return number.ToString();
        }
        var ans = new StringBuilder();
        if (a)
        {
            ans.Append("Pling");
        }
        if (b)
        {
            ans.Append("Plang");
        }
        if (c)
        {
            ans.Append("Plong");
        }
        return ans.ToString();
    }
}